application/controllers/rabbit_helper.py

```diff
@@ -19,9 +20,16 @@ def _encrypt_message(message_json):
     :return: jwe as String
     """
     log.info('Encrypting JSON message')
-    json_secret_keys = current_app.config['JSON_SECRET_KEYS']
-    encrypter = Encrypter(json_secret_keys)
-    return encrypter.encrypt(message_json)
+
+    if current_app.config['SAVE_SEFT_IN_GCP']:
```

Comment:
Why is this living in the rabbit helper? The communication over rabbit will never use GPG.